[docs] Mention priority support on MUI X docs (#16467)
mui/mui-x: docs/data/introduction/priority-support/priority-support.md

---
title: MUI X - Priority Support
---

# Priority Support [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')

<p class="description">Learn how to use priority technical support for MUI components.</p>

## What is priority support?

Priority Support is a premium service ensuring fast, expert assistance for your critical applications, with expedited troubleshooting and priority handling of inquiries.

> Priority support takes precedence over standard support, which is included by default in paid plans. It is backed by an SLA, whereas standard support is not. Support requests are prioritized in the following order:

Reviewer: > It is backed by an SLA, whereas standard support is not

Technically, we must also describe the service level of the standard support, which we do on https://mui.com/legal/technical-support-sla/

```suggestion
Priority support takes precedence over standard support, which is included by default in paid plans. Support requests are prioritized in the following order:
```

:::info
Community < Pro < Premium Standard (No SLA) < Priority Support (SLA governed)
:::

The SLAs included in priority support are:

- **24 hour response SLA**
  Receive an initial response within 24 business hours.
- **4 hour pre-screening SLA**
  Issues submitted through priority support channels undergo an initial review to validate, and reproduce the issue. This ensures your tickets have the right information for debugging.
- **Issue Escalation**
  Critical issues are escalated to senior engineers for focused attention and faster resolutions.

> You can also check more on this in our [Service Level Agreement for Technical Support](https://mui.com/legal/technical-support-sla/).

Reviewer: ```suggestion
For all the details, please check the [Service Level Agreement for Technical Support](https://mui.com/legal/technical-support-sla/).
```

## Who benefits from priority support?

> Priority support is ideal for teams working on mission-critical applications or handling complex projects that demand rapid resolutions and proactive technical assistance. Priority support covers the following MUI products:

Reviewer: Trying to rephrase the value here. 
More details: https://chatgpt.com/share/67efa77d-228c-8006-a53f-7489e5e1720c

```suggestion
Priority support is tailored for enterprise teams who rely on the MUI ecosystem at scale and want faster turnaround on bug reports, closer collaboration on issue triaging, and coverage across the entire component library.

Priority support covers the following products:
```

- Once the issue is created it will have the following labels: [status: needs triage](https://github.com/mui/mui-x/labels/status%3A%20needs%20triage), [support: commercial](https://github.com/mui/mui-x/labels/support%3A%20commercial), [support: unknown](https://github.com/mui/mui-x/labels/support%3A%20unknown)
- A GitHub action will prompt you to follow an external link leading to a validation app where you need to validate your support key.
- Once your support key is successfully validated the labels in the issue will update to the following: [status: needs triage](https://github.com/mui/mui-x/labels/status%3A%20needs%20triage), [support: commercial](https://github.com/mui/mui-x/labels/support%3A%20commercial), [support: priority](https://github.com/mui/mui-x/labels/support%3A%20priority)
- The GitHub action will update the previous comment with the following: `Thank you for verifying your support key 🔑, your SLA starts now.` Thus marking the start of the SLA.

> - Then the support team will check the issue and respond within the SLA.

Reviewer: ```suggestion
- You get a response from the team within the SLA.
```

- Once the issue is created it will have the following labels: [status: needs triage](https://github.com/mui/mui-x/labels/status%3A%20needs%20triage), [support: commercial](https://github.com/mui/mui-x/labels/support%3A%20commercial), [support: unknown](https://github.com/mui/mui-x/labels/support%3A%20unknown)
- A GitHub action will prompt you to follow an external link leading to a validation app where you need to validate your support key.
- Once your support key is successfully validated the labels in the issue will update to the following: [status: needs triage](https://github.com/mui/mui-x/labels/status%3A%20needs%20triage), [support: commercial](https://github.com/mui/mui-x/labels/support%3A%20commercial), [support: priority](https://github.com/mui/mui-x/labels/support%3A%20priority)
- The GitHub action will update the previous comment with the following: `Thank you for verifying your support key 🔑, your SLA starts now.` Thus marking the start of the SLA.
- You get a response from the team within the SLA.

### Through email

> If you need to share **private information** you can [submit a request](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email to [x@mui.com](mailto:x@mui.com) with your support key.

Author: Just checking if this is the correct email ID we want to keep here?

Reviewer: actually, let's remove the email.  Let's stay with the channels mentioned on: https://mui.com/legal/technical-support-sla/#support-channels
